[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-zope.security",
	"source": "zope.security",
	"version": "5.1.0-1",
	"section": "universe/zope",
	"priority": "extra",
	"architecture": "amd64",
	"depends": [
		"python3 (<< 3.9)",
		"python3 (>= 3.8~)",
		"python3-pkg-resources",
		"python3-zope.component",
		"python3-zope.i18nmessageid",
		"python3-zope.interface",
		"python3-zope.location",
		"python3-zope.proxy",
		"python3-zope.schema",
		"python3:any",
		"libc6 (>= 2.4)"
	],
	"conflicts": [
		"zope3"
	],
	"installedSize": 595,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Zope Security Framework",
	"homepage": "http://pypi.python.org/pypi/zope.security",
	"filename": "pool/universe/z/zope.security/python3-zope.security_5.1.0-1_amd64.deb",
	"size": 81444,
	"md5": "8d2c7f2c6a7dee962bd0cf28d4168777",
	"sha1": "d26c86a91dcef6ef2b7f9ad737a52fd7104b0d40",
	"sha256": "69d965ec7204facd9b2117c81bf1fd012931f54b45f3a1e4c744ff82416ef5ae",
	"descriptionMd5": "e9f54e364b11459c8fb9ffe7642eb3bf"
}]